package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "clean": "rm -rf build",
     "build": "npm run clean; tsc",
-    "test": "TZ=utc mocha"
+    "test": "TZ=utc mocha",
+    "device": "npm run build; node build/Device.js",
```

Comment:
Please remove these scripts again. They might be good for your personal use but not fit all. A as Kai device is just kind of an example.